ext/cld3/extconf.rb

```diff
@@ -17,8 +17,14 @@
 require "mkmf"
 
 # Check pkg-config first to inform the library is missing if so.
-unless pkg_config("protobuf") || have_library("protobuf")
-  abort "Can't find protobuf\n"
+def have_protobuf?
+  checking_for("protobuf by pkg-config") do
```

Comment:
Unfortunately, checking_for is "internal use only." Please remove.
https://github.com/ruby/ruby/blob/7d4cdab8f8c7e1f3f7e5c2add712659057f643fc/lib/mkmf.rb#L940